modules/ipi-install-retrieving-the-openshift-installer.adoc

```diff
@@ -6,11 +6,21 @@
 [id="retrieving-the-openshift-installer_{context}"]
 = Retrieving the {product-title} installer
 
-Use the `stable-4.x` version of the installer to deploy the generally
+Use the `stable-4.x` version of the installer based on your selected architecture to deploy the generally
 available stable version of {product-title}:
 
+.For x86_64
 [source,terminal,subs="attributes"]
 ----
 $ export VERSION=stable-{product-version}
+$ export RELEASE_ARCH=aarch64
```

Comment:
If this section is for x86_64, shouldn't this be `RELEASE_ARCH=x86_64`?

Reply:
Yes! I saw that after my initial push, will update both. Thank you